src/main/java/ch/sbb/polarion/extension/pdf/exporter/rest/controller/ConfigurationInternalController.java

```diff
@@ -64,4 +64,12 @@ public class ConfigurationInternalController {
     public @NotNull ConfigurationStatus checkWeasyPrint() {
         return ConfigurationStatusUtils.getWeasyPrintStatus();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/configuration/weasyprint-service-version")
+    @Operation(summary = "Checks WeasyPrint Service configuration")
+    public @NotNull ConfigurationStatus checkWeasyPrintServiceVersion() {
```

Comment:
for what reason we need 2 identical methods? I mean `checkWeasyPrint` and `checkWeasyPrintServiceVersion`